recipes/repeatfinder/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = 1.1 %}
+
+package : 
+  name : repeatfinder
+  version : {{ version }}
+
+build:
+  number: 0
+
+source: 
+  url: https://github.com/jaroslav-kubin/repeatfinder/releases/download/v{{version}}/repeatfinder-{{version}}.tar.gz
+  sha256 : 80fdbff75ea1eced25cfa95baa942f05ce830949237faf954cd37d2a4b6a449d
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+about:
+  home: https://gitlab.fi.muni.cz/xkubin/bachelors-thesis.git
+  license: MUNI
```

Comment:
please change this to the actual license (https://github.com/jaroslav-kubin/repeatfinder/blob/main/LICENSE)